paimon-web-api/pom.xml

```diff
@@ -63,6 +64,12 @@ under the License.
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>com.alibaba</groupId>
+            <artifactId>fastjson</artifactId>
```

Comment:
Why use fastjson? IMO, fastjson has some security problem which is not recomeneded.